### DISCUSSION TITLE
Symbolset for drawing and editing tutorial

### DISCUSSION THREAD

**Discussion by mlerjen** [OpenOrienteering/mapper#1858] [Show and tell]
For a while I am pondering on a way to cover the powerful draw and edit palette Mapper offers to the user.
I so far developed a symbol set to describe the various possible actions graphically. Below a sample. Mapper file with symbolset attached as .zip
[Symbolset Tutorial Drawing with Mapper.zip](https://github.com/OpenOrienteering/mapper/files/5855289/Symbolset.Tutorial.Drawing.with.Mapper.zip)

![image](https://user-images.githubusercontent.com/15611653/105482212-5c6f6f00-5ca8-11eb-9c41-00cf42024727.png)

I think in general the graphical language works. What do you think? Would that be somehow helpful?

How could such be implemented here? - For instant I see three approaches.
a) Just integrate the graphical representations into the manual.
b) Make the overview to a poster (to print out) as an appendix to the manual
c) Develop a third sample.omap to be integrated into the Mapper package. Beginners could just open the file and learn to draw and edit properly just while working with that file.

Your thoughts?

**Comment by dg0yt** [comment]
- Starting with a map file integrated into our examples, we could automatically create graphics files for inclusion in the manual, and/or as a separate PDF.
- For each tool, a line of text below the graphics could name the button which replaces the modifier key on Android.
- We might extract the text parts and integrate them in the translation process. The English language is a significant barrier for many people.
